pkgs/development/python-modules/pythondata-software-compiler_rt/default.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+}:
+
+buildPythonPackage rec {
+  pname = "pythondata-software-compiler-rt";
+  version = "2020.04";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "litex-hub";
+    repo = "pythondata-software-compiler_rt";
+    rev = version;
+    hash = "sha256-nE0ZvdlHj4BfbKd1D+AqCHGpVXNMovOIGKMT32WqZ3A=";
+  };
+
+  pythonImportsCheck = [ "pythondata_software_compiler_rt" ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Python module containing data files for compiler_rt software (for use with LiteX";
+    homepage = "https://github.com/litex-hub/pythondata-software-compiler_rt";
+    license = with licenses; [ ];
```

```suggestion
    license = licenses.asl20;
```
see https://github.com/litex-hub/pythondata-software-compiler_rt/blob/master/LICENSE